edb/lib/sys.edgeql

```diff
@@ -187,3 +187,12 @@ sys::get_transaction_isolation() -> sys::TransactionIsolation
     SET force_return_cast := true;
     USING SQL FUNCTION 'edgedb._get_transaction_isolation';
 };
+
+
+CREATE FUNCTION
+sys::get_current_database() -> str
+{
+    # The results won't change within a single statement.
+    SET volatility := 'STABLE';
```

Comment:
The results won't change ever. You might as well say it's immutable.

Reply:
Is it possible to potentially rename a database? Same question for the role.

Reply:
My understanding was that `IMMUTABLE` actually means universally the same, like arithmetic, for example. So same output across databases and across all time.

Reply:
The underlying function is marked as stable, and so must be the wrapper.

Reply:
Right.